targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_NRF52840_DK/PinNames.h

```diff
@@ -262,6 +234,34 @@ typedef enum {
     PullDefault = PullUp
 } PinMode;
 
+#define LED1 p13
+#define LED2 p14
+#define LED3 p15
+#define LED4 p16
+
+#define BUTTON1 p11
+#define BUTTON2 p12
+#define BUTTON3 p24
+#define BUTTON4 p25
+
+#define SPI_PSELMOSI0 P1_13
+#define SPI_PSELMISO0 P1_14
+#define SPI_PSELSS0 P1_12
+#define SPI_PSELSCK0 P1_15
+
+#define SPI_PSELMOSI1 P1_2
+#define SPI_PSELMISO1 P1_3
+#define SPI_PSELSS1 P1_1
+#define SPI_PSELSCK1 P1_4
+
+#define SPIS_PSELMOSI P1_2
+#define SPIS_PSELMISO P1_3
+#define SPIS_PSELSS P1_1
+#define SPIS_PSELSCK P1_4
+
+#define I2C_SDA0 = p26
+#define I2C_SCL0 = p27
```

Comment:
Is having `=` here correct?

Reply:
Obviously not, I will fix it when the test case debate is resolved.

Reply:
Test case will be OK after #15124

Reply:
Fixed